src/Features/Core/Portable/EmbeddedLanguages/Json/JsonOptions.cs

```diff
@@ -0,0 +1,59 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Immutable;
+using System.Composition;
+using Microsoft.CodeAnalysis.Host.Mef;
+using Microsoft.CodeAnalysis.Options;
+using Microsoft.CodeAnalysis.Options.Providers;
+
+namespace Microsoft.CodeAnalysis.EmbeddedLanguages.Json;
+
+internal class JsonOptions
```

Comment:
I don't see where these options are used. Could you link?

Reply:
they're currently not.  i can pull out (they're part of the 'features' work that is coming in in the followup to this PR).

Reply:
ok. it would be better to pull out. The options should be defined in EditorFeatures layer.